source/deploy/mobile-faq.rst

```diff
@@ -372,3 +372,15 @@ In these scenarios, the ``login`` audit log won't have the ``deviceId``, and the
 If the app receives the ``deviceId`` later, the device will send the new ``deviceId`` to the server, generating an ``attachDeviceId`` audit log, and adding the ``deviceId`` to the session data in the database.
 
 Since the ``deviceId`` relates to the application, connections through the web browser, even on mobile, won't have a ``deviceID``.
+
+Where can I find mobile message notification logs?
+-------------------------------------------------------------
+Notification messages are logged to the ``notifications.log`` file. However, this file can only be accessed by server admins.
+Generally you will find it by navigating to ``Notification Settings--->File Location``. One thing to note here is that the ``notifications.log`` exists only if you have enabled it in the Notification Settings i.e., ``NotificationSettings.EnableFile`` should be set to ``TRUE``.
```

Comment:
Where would an admin access ``Notification Settings > File Location`` specifically?

Reply:
Thanks for pointing this out. Will modify this to add better details.